module.c

```diff
@@ -548,6 +548,9 @@ static JSValue js_c_function(
 	JSValue js_result = JS_NULL;
```

Comment:
(looks like we're missing a `end_call_python(context);` next to `return JS_ThrowInternalError(ctx, "Internal error: could not convert args.");`, btw)

Reply:
You're right. It's an "internal error" so I can not trigger it in a test. Important to fix though, as we otherwise risk deadlocking the interpreter.